libraries/chain/include/eosio/chain/hotstuff.hpp

```diff
@@ -68,6 +68,13 @@ namespace eosio::chain {
       eosio::chain::quorum_certificate current_qc;
       eosio::chain::extended_schedule schedule;
       map<fc::sha256, hs_proposal_message> proposals;
+
+      hs_proposal_message* get_proposal(const fc::sha256 &id) {
```

Comment:
Not a huge deal, but please to follow current code convention of `const type& arg`